src/features/municipality/ui/components/MunicipalityBoardsMap.tsx

```
@@ -0,0 +1,318 @@
+"use client";
+
+import "mapbox-gl/dist/mapbox-gl.css";
+
+import { Box, Tooltip, Typography } from "@mui/material";
+import * as turf from "@turf/turf";
+import type { MutableRefObject } from "react";
+import { useCallback, useEffect, useMemo, useRef } from "react";
+import Map, { Layer, MapRef, Marker, Source } from "react-map-gl/mapbox";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)